pkg/storaged/content-views.jsx

```diff
@@ -80,17 +78,25 @@ function create_tabs(client, target, is_partition) {
     var is_filesystem = (block && block.IdUsage == 'filesystem');
     var is_crypto = (block && block.IdUsage == 'crypto');
 
+    var warnings = client.path_warnings[target.path] || [ ];
+
     var tabs = [ ];
     var row_action = null;
 
-    function add_tab(name, renderer) {
+    function add_tab(name, renderer, associated_warnings) {
+        let tab_warnings = [ ];
+        if (associated_warnings)
+            tab_warnings = warnings.filter(w => associated_warnings.indexOf(w.warning) >= 0);
+        if (tab_warnings.length > 0)
+            name = <span><span className="pficon pficon-warning-triangle-o" /> {name}</span>;
```

Comment:
Is that extra space before `{name}` deliberate, or a typo?

Reply:
It's deliberate and produces a space between the triangle and the name.  Otherwise they run together.